transaction/src/model/instruction.rs

```diff
@@ -35,19 +35,14 @@ pub enum Instruction {
         bucket_id: ManifestBucket,
     },
 
-    /// Asserts worktop contains resource.
-    AssertWorktopContains {
-        resource_address: ResourceAddress,
-    },
-
     /// Asserts worktop contains resource by at least the given amount.
-    AssertWorktopContainsByAmount {
+    AssertWorktopContains {
```

Comment:
What are your thoughts on also fixing the order of the parameters on all these instructions so that `resource_address` is first, as per the transfer methods and other changes?

I thought generally we agreed a few months ago we were going to change all instructions to have ResourceAddress first, because as you write these instructions, you care about what resource, and then the amount.

(Appreciate you might say "assert I have 5 XRD" / "transfer 5 XRD" but we said in each case specifying Transfer XRD 5 was more logical :) )

Reply:
As commented in another PR. I don't mint either way. A larger clean-up will be required for the parameter order change.